rest-api-spec/src/main/resources/rest-api-spec/api/text_structure.find_message_structure.json

```diff
@@ -0,0 +1,98 @@
+{
+  "text_structure.find_message_structure":{
+    "documentation":{
+      "url":"https://www.elastic.co/guide/en/elasticsearch/reference/current/find-message-structure.html",
+      "description":"Finds the structure of a list of messages. The messages must contain data that is suitable to be ingested into Elasticsearch."
+    },
+    "stability":"stable",
+    "visibility":"public",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/json"]
+    },
+    "url":{
+      "paths":[
+        {
+          "path":"/_text_structure/find_message_structure",
+          "methods":["GET", "POST"]
+        }
+      ]
+    },
+    "params":{
+      "lines_to_sample":{
+        "type":"int",
+        "description":"How many lines of the file should be included in the analysis",
+        "default":1000
+      },
+      "line_merge_size_limit":{
+        "type":"int",
+        "description":"Maximum number of characters permitted in a single message when lines are merged to create messages.",
+        "default":10000
+      },
```

Comment:
Similar to the other endpoint, if the input is a list of messages then the code shouldn't need to merge lines into messages so this parameter is redundant.